packages/office-ui-fabric-react/src/index.ts

```diff
@@ -89,4 +89,32 @@ export * from './Utilities';
 export * from './Viewport';
 export * from './WindowProvider';
 
+export {
```

Comment:
Nit - we could probably export * here since the files have explicit exports.

Reply:
i went with this approach to ensure compat with the v8 file. fewer chances of surprises, or someone exported in v7 that isn't also in v8